web/src/components/ui/MarkdownViewer.tsx

```diff
@@ -76,6 +80,13 @@ const MarkdownImage: Components["img"] = ({ src, alt }) => {
             width={0}
             height={0}
             className="h-full w-full object-contain"
+            onError={(error) => {
```

Comment:
i'd render these inline instead of with a toast. toasts can be spammy if lots of images in a single app fail and are not really exceptions but rather warnings